components/textfield/metadata/textarea.yml

```diff
@@ -1,6 +1,6 @@
 name: Text area
 status: Verified
-description: A multi-line text field (aka textarea)
+description: A multi-line text field.
```

Comment:
I think it could still be important to mention that it is in fact a text area since the html is using `<textarea>` and not `<input>` but perhaps in a more polished way. 

"A multi-line text field, using the text field element" 

And if we're updating it here do we also want to update the description in textfield?

Reply:
Ohh I didn't realize the descriptions were the same for both Text Field and Text Area. Good catch, I'll update 👍